File: work/nh/projects/nh_it_apps/api/Models/Entities/NetworkInterface.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models.Entities
{
    public class NetworkInterface : BaseEntity
    {
        public int ServerId { get; set; }

        [Column(TypeName="varchar(20)")]
        public string Type { get; set; }

        [Column(TypeName="varchar(20)")]
        public string MacAddress { get; set; }
        
        public IEnumerable<IP> IPs { get; set; }
    }
}

File: work🏢/nh/projects/dev/nh_it_apps/client/src/app/models/NetworkInterface.ts
import { IP } from "./IP";

export class NetworkInterface {
    id: number = 0;
    serverId : number = 0;
    type: string = "";
    macAddress: string = "";
    comments: string = "";
    lastUpdatedBy: string = "";
    lastUpdated: Date = new Date();
    iPs: IP[] = [];
}